procedures/pulpcore/update.md.erb

```diff
@@ -1,14 +1,14 @@
 # Prepare the build environment
 
-* [ ] Clone https://github.com/evgeni/pulpcore-packaging
-* [ ] [Configure](https://github.com/evgeni/pulpcore-packaging#configure) the build environment
-* [ ] [Build](https://github.com/evgeni/pulpcore-packaging#build) the container
+* [ ] Clone https://github.com/theforeman/pulpcore-packaging
+* [ ] [Configure](https://github.com/theforeman/pulpcore-packaging/automation#configure) the build environment
+* [ ] [Build](https://github.com/theforeman/pulpcore-packaging/automation#build) the container
```

Comment:
Running through the steps, I did not find it mentioned that I needed to cd into the automation directory.  Not sure if that belongs here or in the `pulpcore-packaging/automation/README.md`

Reply:
https://github.com/theforeman/pulpcore-packaging/pull/42